filters/openpolicyagent/openpolicyagent.go

```diff
@@ -764,6 +867,28 @@ func (opa *OpenPolicyAgentInstance) DistributedTracing() opatracing.Options {
 	return buildTracingOptions(opa.registry.tracer, opa.bundleName, opa.manager)
 }
 
+func (opa *OpenPolicyAgentInstance) startPluginTriggerDaemon() {
+	interval := 10 * time.Second
```

Comment:
Shouldn't this be configurable?

Reply:
for sure, unless we have a sensible default